contributing/chestertons-fence.md

```diff
@@ -0,0 +1,77 @@
+# Chesterton's Fence
+
+Hello :wave:
+You're probably here becomes someone directed you to this post in response to you suggesting an idea about how Exercism could be improved.
+Before our team invest time into exploring your idea, they're hoping you'll read this and consider whether your idea might of being thought of before, what pitfalls it might contain.
+**By adding those problems and pitfalls to your suggestion, and considering why your idea might not have already been implemented, you are likely to get a faster and more positive response.**
+
+## So what does "Chesterton's Fence" mean?
+
+Chesterton’s Fence is an idea inspired by a quote from the writer G.K. Chesterton’s 1929 book, The Thing.
+It became well-known as it was quoted by John F. Kennedy.
+This is the original quote:
+
+> There exists in such a case a certain institution or law; let us say, for the sake of simplicity, a fence or gate erected across a road.
+> The more modern type of reformer goes gaily up to it and says, “I don’t see the use of this; let us clear it away.”
+> To which the more intelligent type of reformer will do well to answer: “If you don’t see the use of it, I certainly won’t let you clear it away. Go away and think. Then, when you can come back and tell me that you do see the use of it, I may allow you to destroy it."
+
+The idea of Chesterton’s Fence is that you if don't understand why something is there, you probably don't understand why it should be removed.
+Equally, if you don't understand why something **isn't** there, you probably don't understand why it's been left out.
```

Comment:
I also really like the highlighted sentence on https://fs.blog/2020/03/chestertons-fence/. 

> Do not remove a fence until you know why it was put up in the first place.

Perhaps that's worth mentioning as a simple rule/goal to follow.

Reply:
Added :)